[Documentation] Behaviour Feedback widgets description (#6661)
aces/Loris: modules/bvl_feedback/README.md

# Behavioural Feedback

## Configurations

> The Feedback module allows users to flag and comment on  behavioural database fields.

Reviewer: The first paragraph is still the purpose, it's just the second that I was saying is a configuration

> The Feedback module allows users to flag and comment on any database field particularly for behavioural data entry. 

Reviewer: ```suggestion
The Feedback module allows users to flag and comment on field behavioural database fields. 
```

> Two default feedback types exist: input and scoring errors. Additional feedback types can be added to feedback_bvl_type table.

Reviewer: This should be under a `## Configurations` section, not the purpose.

## Behavioral Feedback Notifications

> The Dashboard Widget tracks issues and corrections to behavioural/clinical data points. Notification are clickable and redirect to the instrument with new changes or feedback.

Reviewer: This should be in an interactions section describing how it registers widgets on the dashboard, the description of being clickable is unnecessary for the spec. It's an incidental part of the implementation, not a core part of the design.
